src/agents/agent.py

```diff
@@ -63,7 +64,16 @@ def search_queries(self, queries: list, project_name: str) -> dict:
         results = {}
         
         knowledge_base = KnowledgeBase()
-        bing_search = BingSearch()
+        user_search_engine = Config().get_search_engine()
+
+        if user_search_engine == "DDGS":
+            search_engine = DuckDuckGoSearch()
+        else:
+            """
+            Set default search engine to Bing
+            """
+            search_engine = BingSearch()
```

Comment:
**Suggestion**: Refactor the conditional logic for selecting the search engine to a separate method or function. This change would improve code readability and maintainability, especially if more search engines are added in the future. <br> **Code Suggestion**: 
 ```
 +        search_engine = self.get_search_engine(user_search_engine)
 ```